__tests__/integration.test.js

```diff
@@ -55,6 +55,32 @@ describe("GET /api/topics", () => {
   });
 });
 
+describe("GET /api/articles", () => {
+  test("200: returns an articles array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        for (const article of articles) {
+          if (article.length !== 0) {
```

Comment:
We should also check the length of the array if asserting inside a forEach, to ensure we get all articles back.